device/rsvp/rsvpTspec.yaml

```diff
@@ -0,0 +1,42 @@
+components:
+  schemas:
+    Rsvp.Tspec:
+      description: >-
+        Configuration for RSVP-TE TSPEC object included in Path Messages. The usage of these parameters is defined in RFC2215.
+      type: object
+      properties:
+        token_bucket_rate:
+          description: >-
+            The rate of the traffic to be carried in this LSP in bytes per second.
+            This is part of the Token Bucket specification defined for a traffic flow defined in RFC2215.
```

Comment:
does 0 have special meaning?  some indication on the unit for the rate w/o looking up RFC2215 will help.

Reply:
The specification doesn't disallow any value.It is upto the Dut to honour or dishonour. Dut dishonours by sending a RESV Err.One specification of Cisco, Cisco IOS-XR : https://www.cisco.com/c/en/us/td/docs/ios-xml/ios/mp_te_path_setup/configuration/xe-16-8/mp-te-path-setup-xe-16-8-book/mpls-traffic-engineering-automatic-bandwidth-adjustment-for-te-tunnels.pdf
The bandwidth argument is the bandwidth, in kilobits persecond, set for the MPLS TE tunnel. The range is from 1 to 4294967295. The default is 0.This is not totally according to spec.

Reply:
you mentioned the bandwidth is in kilobits/sec in the comment, but the description is "bytes per second"?  so which is it?

Reply:
The unit for the rate as defined in the specification is bytes/sec.  The comment above was mainly providing an existing example in a DUT on how this configuration is opened up to user ( in IOS-XR ) .   In the model  , we have tried to keep it as per specification , which is in bytes/sec.